### DISCUSSION TITLE
[Art competition] Incompleteness

### DISCUSSION THREAD

**Discussion by Calmadeas** [prosyslab-classroom/cs424-program-reasoning#258] [Art competition]
![DALL·E](https://user-images.githubusercontent.com/90555950/204805403-ba9cb6f5-201b-42e4-ad51-57b5c0c9c966.png)

## Short Description
For me, [Incompleteness] means [A beautiful little rose in a shattered vase with grotesque root shredding floor tiles and a falling person, digital art]".

## Long Description (up to 3 sentences)
What we want was a beautiful little rose in a vase: the perfect arithmetic of natural numbers. However, the rose grew to break our vase and even the floor we are standing: the solid belief "The truth exists, so does a proof, the rationality we are proud of will eventually reveal them all."

**Comment by KihongHeo** [comment]
Grotesque. But I am happy to see the rose is still alive and looks very strong.
